app/src/main/java/nyc/c4q/jonathancolon/inContaq/contactlist/fragments/ContactStatsFragment.java

```diff
@@ -37,18 +41,13 @@
 import static com.db.chart.renderer.AxisRenderer.LabelPosition;
 
 //This fragment handles graph logic
-public class ContactStatsFragment extends Fragment {
-
-    /*
-    These color constants are hardcoded for backwards compatibility.
-     */
+public class ContactStatsFragment extends Fragment implements AdapterView.OnItemSelectedListener {
 
     private static final String BLUE_SAPPHIRE = "#0E587A";
```

Comment:
Better to name these with their usage in the app (can comment with color names if you want)